tool_quality_checks/importer.py

```diff
@@ -30,14 +30,25 @@ def import_sewerage_data_into_db(db, settings):
     import_file_based_on_filetype(
         db, settings, settings.manhole_layer, "putten_" + settings.import_type
     )
-    import_file_based_on_filetype(
-        db, settings, settings.pipe_layer, "leidingen_" + settings.import_type
-    )
+
+    # check if pipe_layer is available
+    if hasattr(settings, "pipe_layer"):
+        import_file_based_on_filetype(
+            db, settings, settings.pipe_layer, "leidingen_" + settings.import_type
+        )
+        available_gbi_pipe_layer = True
```

Comment:
`available_gbi_pipe_layer` = de beschikbare pijp-laag (taalkundig).
`gbi_pipe_layer_available` zou duidelijker zijn. Of, nog meer volgens conventies, `has_gbi_pipe_layer`. Duidelijk dat het een ja/nee is op die manier.